chainer/functions/noise/gumbel_softmax.py

```diff
@@ -0,0 +1,34 @@
+from chainer import cuda
+from chainer.functions.activation.softmax import softmax
+
+
+def gumbel_softmax(log_pi, tau=0.1, axis=1):
```

Comment:
Why not using `Variable` for `tau`?

Reply:
Maybe I'm misunderstanding but I couldn't get your point.
In my understanding, when `Variable` is passed, it can be properly backproped because current implementation is just combination of arithmetic and `Function` call.
If backprop is not needed scalar is enough.